chapter4/e3-bean-counting.js

```diff
@@ -0,0 +1,15 @@
+function countBs(string) {
+  let numBs = 0;
+  for (i = 0; i < string.length; i++) {
+    if (string[i] == "B") numBs++;
```

Comment:
Always use curly braces for control structures. It makes the visual groupings more apparent and saves you from a surprisingly common long term bug where someone adds another line and doesn't realize it's in/not in the if/for structure.